clang/lib/Sema/SemaARM.cpp

```diff
@@ -1329,6 +1329,11 @@ void SemaARM::handleInterruptAttr(Decl *D, const ParsedAttr &AL) {
     return;
   }
 
+  const TargetInfo &TI = getASTContext().getTargetInfo();
+  if (TI.hasFeature("vfp")) {
+    Diag(D->getLocation(), diag::warn_arm_interrupt_vfp_clobber);
+  }
```

Comment:
https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements

Reply:
Huh, I guess the rule has exceptions so the clang-format checker doesn't flag it? Will fix.

Reply:
I think clang-format will very rarely add or remove tokens, just shuffle them around. So unfortunately this "rule" is just a when we see it kinda thing.

Reply:
Oh, right. Well, fixed now.